### PR TITLE
Add in Ticket ref from branch name to each commit

### PR DESCRIPTION
**Pull request by greg0ire**
Closes #51

### CODE REVIEW

**Comment by greg0ire** [comment]
I used shellcheck and tweaked the documentation, please review @davidcollom @Edo76

**Comment by Edo78** [comment]
It looks fine to me

File: tests/ticketref_test.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+testRefIsAddedasSuffixToCommit()
+{
+	testRepo=$SHUNIT_TMPDIR/test_repo3
+	mkdir -p "${testRepo}"
+	cd "${testRepo}" || exit 1
+	git init --quiet .
+	git config hooks.enabled-plugins ticketref
+  git config hooks.ticketref.position "POST"
```
Comment (greg0ire) [inline tests/ticketref_test.sh]: Woops, didn't notice this

**Comment by davidcollom** [comment]
Looks good to me! 👍

**Comment by greg0ire** [comment]
Merged, thanks a lot @davidcollom !

**Pull request merged**